src/app/pages/TopicPage/getComponent.js

```diff
@@ -0,0 +1,27 @@
+import { VISUAL_PROMINENCE, VISUAL_STYLE, COMPONENT_NAMES } from './constants';
+
+const { NONE, BANNER } = VISUAL_STYLE;
+const { NORMAL, HIGH } = VISUAL_PROMINENCE;
+const {
+  MESSAGE_BANNER,
+  SIMPLE_CURATION_GRID,
+  HIERARCHICAL_CURATION_GRID,
+  BILLBOARD,
+} = COMPONENT_NAMES;
+
+export default (visualStyle, visualProminence) => {
```

Comment:
```suggestion
export default (visualStyle, visualProminence) => {
  return {
    "BANNER_NORMAL": "MESSAGE_BANNER",
    "BANNER_HIGH": "BILLBOARD",
    "NONE_NORMAL": "SIMPLE_CURATION_GRID",
    "NONE_HIGH": "HIERARCHICAL_CURATION_GRID",
  }[`${visualStyle}_${visualProminence}`] || null;
}
```

Reply:
LGTM.
Please consider the rewrite above to eliminate the
branching from if statements. 😄

Reply:
Made this change with a slight difference suggested by @karinathomasbbc to make it easier to read. Compared computational efficiency of each way of doing it and accessing data from objects is O(1) apparently! So this is a good way of doing it compared to the presumably O(n) of multiple unbroken-out-of if clauses to evaluate.